app/src/main/java/chat/rocket/android/server/infraestructure/SharedPreferencesMessagesRepository.kt

```diff
@@ -13,6 +13,28 @@ class SharedPreferencesMessagesRepository(
     private val moshi: Moshi,
     private val currentServerInteractor: GetCurrentServerInteractor
 ) : MessagesRepository {
+    private val KEY_LAST_SYNC_DATE = "KEY_LAST_SYNC_DATE"
+
+    override suspend fun saveLastSyncDate(currentTimeMillis: Long) {
+        withContext(CommonPool) {
+            currentServerInteractor.get()?.let {
+                prefs.edit().putLong(getSyncDateKey(it), currentTimeMillis).apply()
+            }
+        }
+    }
+
+    override suspend fun getLastSyncDate(): Long? = withContext(CommonPool) {
+        currentServerInteractor.get()?.also { server ->
+            if (!prefs.contains(getSyncDateKey(server)))
+                return@withContext null
+            //
```

Comment:
It's supposed to be a comment? 
Could you please remove that or update that?

Reply:
yep, removed